src/webgpu/api/validation/image_copy/buffer_texture_copies.spec.ts

```diff
@@ -304,3 +304,43 @@ g.test('depth_stencil_format,copy_buffer_offset')
       unreachable();
     }
   });
+
+g.test('sample_count')
+  .desc(
+    `
+  Test that the texture sample count. Check that a validation error is generated if sample count is
+  not 1.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('sampleCount', [1, 4])
+      .beginSubcases()
+      // writeTexture is handled by writeTexture.spec.ts.
+      .combine('copyType', ['CopyB2T', 'CopyT2B'] as const)
```

Comment:
µNit: I'd flip these:

```suggestion
      // writeTexture is handled by writeTexture.spec.ts.
      .combine('copyType', ['CopyB2T', 'CopyT2B'] as const)
      .beginSubcases()
      .combine('sampleCount', [1, 4])
```

Because if you were looking at test results, it would be useful to see, for example, that CopyB2T fails when CopyT2B doesn't. But if the validation were missing then sampleCount 1 would still be expected to pass so it's not very useful to look at the results for 1 vs 4 side-by-side.

Reply:
Thank you for pointing it out. Fixed.